recipes/glpk/meta.yaml

```diff
@@ -1,23 +1,27 @@
-# copied from https://anaconda.org/mcg/glpk 
+# copied and modified from https://anaconda.org/conda/conda-recipes
 
 package:
   name: glpk
-  version: 4.56
+  version: 4.57
 
 source:
-  fn: glpk-4.56.tar.gz
-  url: http://ftp.gnu.org/gnu/glpk/glpk-4.56.tar.gz
-
-requirements:
-  build:
-  run:
+  fn: glpk-4.57.tar.gz
+  url: http://ftp.gnu.org/gnu/glpk/glpk-4.57.tar.gz
+  md5: 237531a54f73155842f8defe51aedb0f
 
 build:
+  features:
+    - vc9      [win and py27]
+    - vc10     [win and py34]
+    - vc14     [win and py35]
+
   number: 0
 
+requirements:
+  build:
+    - gmp      [not win]
```

Comment:
We also need gcc as a build dependency and libgcc as a run dependency here. We don't build for win, so you could elimininate the windows stuff for simplicity. But that's up to you. Further, according to the [docs](http://conda.pydata.org/docs/building/meta-yaml.html#skipping-builds), it is best practice to have a ``#`` in front of the selectors.